packages/forklift-console-plugin/src/modules/Providers/utils/helpers/missingKeysInSecretData.ts

```diff
@@ -21,7 +21,7 @@ export function missingKeysInSecretData(secret: IoK8sApiCoreV1Secret, keys: stri
     const secretValue = secret.data[key] && Base64.decode(secret.data[key]);
 
     // Check if the key exists and is not null or empty string
-    if (!secretValue || secretValue.trim() === '') {
+    if (!secretValue || secretValue?.trim() === '') {
```

Comment:
```suggestion
    if (!secretValue || secretValue.trim() === '') {
```
`secretValue` can't be null because of `!secretValue` , and we know it's a string because we created it in line 21, what does adding optional chaining operator do here, what am i missing ?